apps/backoffice-v2/src/domains/business-reports/fetchers.ts

```diff
@@ -98,7 +105,7 @@ export const fetchBusinessReports = async ({
   const queryParams = qs.stringify(
     {
       ...params,
-      type: reportType,
+      ...(reportType !== 'All' && { reportType }),
```

Comment:
The fetcher is relatively low level. Filtering `All` ideally should occur much before, for example, in the component logic.